config/site_settings.yml

```diff
@@ -2036,6 +2036,10 @@ developer:
     default: false
     client: true
     hidden: true
+  experimental_hashtag_search_result_limit:
```

Comment:
Will probably get rid of this later, mainly put it in there for quick feedback loops of experimentation